src/EVPFFT/scripts/setup-env.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash -e
+show_help() {
```

Comment:
this is the same args as the build-it.sh. Please lets avoid code repetition as it would make it harder to maintain this scripts